website/source/api/secret/identity/group-alias.html.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: "api"
+page_title: "Identity Secret Backend: Group Alias - HTTP API"
+sidebar_current: "docs-http-secret-identity-group-alias"
+description: |-
+  This is the API documentation for managing the group aliases in the identity store.
+---
+
+## Create a Group Alias
+
+This endpoint creates or updates a group alias.
+
+| Method   | Path                     | Produces               |
+| :------- | :----------------------- | :----------------------|
+| `POST`   | `/identity/group-alias`  | `200 application/json` |
+
+### Parameters
+
+- `name` `(string: entity-<UUID>)` – Name of the group alias.
+
+- `id` `(string: <optional>)` - ID of the group alias. If set, updates the
+  corresponding existing group alias.
+
+- `mount_accessor` `(string: "")` – Mount accessor to which this alias belongs
+  toMount accessor to which this alias belongs to.
+
+- `canonical_id` `(string: "")` - ID of the group to which this is an alias.
+
+
+### Sample Payload
+
+```json
+{
+	"canonical_id": "b86920ea-2831-00ff-15c5-a3f923f1ee3b",
```

Comment:
Is this nested under the data block?

Reply:
No. This is the payload which is not nested under any block, unless I am misunderstanding your comment.